plugin/trino-ignite/src/test/java/io/trino/plugin/ignite/TestIgniteClient.java

```diff
@@ -59,11 +67,36 @@
 
     public static final JdbcClient JDBC_CLIENT = new IgniteClient(
             new BaseJdbcConfig(),
-            session -> { throw new UnsupportedOperationException(); },
+            new TestRetryingConnectionFactory.MockConnectorFactory(
+                    TestRetryingConnectionFactory.MockConnectorFactory.Action.RETURN),
```

Comment:
need this change as there is 1 call to openConnection(session) which was failing without this mock.